templates/skeleton/app/routes/account.orders.$id.tsx

```diff
@@ -100,7 +129,7 @@ export default function OrderRoute() {
                 <p>Subtotal</p>
               </th>
               <td>
-                <Money data={order.subtotalPriceV2!} />
+                <Money data={order.subtotal!} />
```

Comment:
If the API has these values as nullable... I wonder if we should update Money to allow nullable as well and then throw a proper error instead of requiring `!` everywhere 🤔 

cc @juanpprieto thoughts?

That or update our code to `{order.subtotal && <Money ... />}` everywhere 🤔